test/views/Delta.spec.js

```diff
@@ -226,6 +229,181 @@ describe('Delta', function () {
       });
 
     });
+
+    describe('using groupedTimeSeriesData', function () {
```

Comment:
Yeah why not I'll do that now. I was just trying to avoid the tests being to massiveeee.

On Thu, Mar 12, 2015 at 9:37 AM, Jon Wyatt <notifications@github.com>
wrote:

>> @@ -226,6 +229,181 @@ describe('Delta', function () {
>>        });
>>  
>>      });
>> +
>> +    describe('using groupedTimeSeriesData', function () {
> Does it need another test(s) to confirm the formatted values are being generated, as well as the keys?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/alphagov/performanceplatform-client.js/pull/47/files#r26288019